src/plugins/discover/public/index.ts

```diff
@@ -16,4 +16,5 @@ export function plugin(initializerContext: PluginInitializerContext) {
 
 export { SavedSearch, SavedSearchLoader, createSavedSearchesLoader } from './saved_searches';
 export { ISearchEmbeddable, SEARCH_EMBEDDABLE_TYPE, SearchInput } from './application/embeddable';
+export { loadSharingDataHelpers } from './shared';
```

Comment:
This will be used server side, right? so I think  `loadSharingDataHelpers` should live in `src/plugins/discover/common`

Reply:
This is just to be used in public code (dashboard panel action: https://github.com/elastic/kibana/pull/88303/files#diff-0154cfbb823111800ea5acf098c44670bcf6035935df851983db9f2c55bfe16cR66